app/controllers/assessment/grading.rb

```diff
@@ -286,8 +286,9 @@ def submission_popover
   end
 
   def score_grader_info
-    grader = Score.find(params[:id]).grader
-    if grader
```

Comment:
@icanberk Apparently the new Rails overrode `params[:id]` with the id of the controller, i.e. assessment.